<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>463</width>
    <height>410</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Параметры запуска</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/resources/configure.png</normaloff>:/resources/configure.png</iconset>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>4</number>
   </property>
   <property name="leftMargin">
    <number>4</number>
   </property>
   <property name="topMargin">
    <number>4</number>
   </property>
   <property name="rightMargin">
    <number>4</number>
   </property>
   <property name="bottomMargin">
    <number>4</number>
   </property>
   <item>
    <widget class="QGroupBox" name="genericBox">
     <property name="title">
      <string>Клиент</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="clientLabel">
        <property name="text">
         <string>Тип клиента</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="clientCombo"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="stateLabel">
        <property name="text">
         <string>Статус</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="stateEdit">
        <property name="text">
         <string/>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="versionLabel">
        <property name="text">
         <string>Требуемая версия</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="versionCombo"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="sizeBox">
     <property name="title">
      <string>Указать размер окна</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QHBoxLayout" name="hzLayout">
      <item>
       <widget class="QLabel" name="widthLabel">
        <property name="text">
         <string>Ширина</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="widthSpinBox">
        <property name="maximum">
         <number>10000</number>
        </property>
        <property name="value">
         <number>854</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Высота</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="heightSpinBox">
        <property name="maximum">
         <number>10000</number>
        </property>
        <property name="value">
         <number>480</number>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="shspc">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="javapathBox">
     <property name="title">
      <string>Указать путь к Java вручную</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="javapathLabel">
        <property name="text">
         <string>Путь к Java</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="javapathEdit"/>
      </item>
      <item>
       <widget class="QToolButton" name="javapathButton">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="argsBox">
     <property name="title">
      <string>Указать дополнительные параметры Java</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="argsLabel">
        <property name="text">
         <string>Параметры</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="argsEdit"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="serviceBox">
     <property name="title">
      <string>Сервис</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QPushButton" name="opendirButton">
        <property name="text">
         <string>Открыть каталог клиента...</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="hspc">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="saveButton">
        <property name="text">
         <string>Сохранить</string>
        </property>
        <property name="default">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>clientCombo</tabstop>
  <tabstop>versionCombo</tabstop>
  <tabstop>stateEdit</tabstop>
  <tabstop>javapathBox</tabstop>
  <tabstop>javapathEdit</tabstop>
  <tabstop>javapathButton</tabstop>
  <tabstop>argsBox</tabstop>
  <tabstop>argsEdit</tabstop>
  <tabstop>opendirButton</tabstop>
  <tabstop>saveButton</tabstop>
 </tabstops>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
